### DISCUSSION TITLE
I am interested in installing qBittorrent but...

### DISCUSSION THREAD

**Discussion by Simpuhl** [tteck/Proxmox#2633] [Q&A] [answered]
I am using the site and the command to install it is:
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/qbittorrent.sh)"

However, I am wondering if I can install a specific version? I want 4.6.3, I am not familiar enough with the LCX containers so I don't know if its possible, I know I can pull specific images in docker though.

Thank you

**Comment by tteck** [comment, marked as answer]
The script utilizes the Debian package manager to install qbittorrent-nox, 
https://github.com/tteck/Proxmox/blob/b12a98e5a3e647217bb481342041ae9ea6958eb1/install/qbittorrent-install.sh#L23
which the current version is 4.5.2 https://packages.debian.org/bookworm/amd64/qbittorrent-nox/download

**Comment by Simpuhl** [comment]
Hi @tteck 

Thank you for your response,

I was actually able to upgrade myself.

The steps I took if anybody is curious:

I went to:  
`/usr/bin`
Renamed qbittorrent-nox to qbittorrent-noxold using: 
`mv qbittorrent-nox qbittorrent-noxold`
Downloaded the new version from https://github.com/userdocs/qbittorrent-nox-static using wget, I specifically needed x86_64: 
`wget -qO ./qbittorrent-nox https://github.com/userdocs/qbittorrent-nox-static/releases/latest/download/x86_64-qbittorrent-nox`
Finally I ran: 
`chmod 700 ./qbittorrent-nox`
Rebooted lxc container, started just fine with same configs

The latest version just happens to be 4.6.3 also.

Maybe you can utilize those package instead? or make a separate version of the script? It's all up to you of course, but I think others may like the option also.

Appreciate all you do.
